apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -151,11 +151,11 @@ type FlowCollectorIPFIX struct {
 // Agent feature, can be one of:<br>
 // - `PacketDrop`, to track packet drops.<br>
 // - `DNSTracking`, to track specific information on DNS traffic.<br>
-// - `FlowRTT`, to track TCP latency [Unsupported (*)].<br>
-// - `NetworkEvents`, to track Network events.<br>
-// - `PacketTranslation`, to enrich flows with packets translation information. <br>
-// - `EbpfManager`, to enable using EBPF Manager to manage netobserv ebpf programs [Developer Preview].<br>
-// - `UDNMapping`, to enable interfaces mapping to udn [Developer Preview]. <br>
+// - `FlowRTT`, to track TCP latency.<br>
+// - `NetworkEvents`, to track network events [Technology Preview].<br>
+// - `PacketTranslation`, to enrich flows with packets translation information, such as Service NAT.<br>
+// - `EbpfManager`, to enable using eBPF Manager to manage NetObserv eBPF programs. [Unsupported (*)].<br>
```

Comment:
This is also Technology Preview right?

Reply:
iirc we're indeed calling it TP internally but for downstream doc it's not mentioned as such to avoid some additional hassle. @skrthomas do you confirm?
Network Events feature is different because it _is_ also a TP considered from core ocp point of view.